### ISSUE TITLE
Add CGMES v3 support

### ISSUE THREAD

**Issue by m-mirz** [cim-iec/cimpy#7]
The CGMES v3 seems to be available [here](https://www.entsoe.eu/digital/cim/cim-for-grid-models-exchange/) but the schema files are still missing.

**Comment by chicco785** [comment]
@m-mirz is there any reason why v3 version is not yet supported? I think rdf profiles can now be found around :)

**Comment by dinkelbachjan** [comment]
We have the CGMES v3 RDFS documents now available [here](https://github.com/sogno-platform/cimgen/tree/7907250da3c430077c7a3c88673e63ab1553ef8e/cgmes_schema/CGMES_3.0.0) and intend to incorporate the corresponding Python classes generated by CIMgen into CIMpy

**Comment by iripiri** [comment]
The Python files for CGMES v3 are uploaded in a separate branch, open TODOs:
- [ ] adapt documentation
- [ ] add/adapt examples
- [ ] adapt cimexport.py
- [ ] add/adapts tests

**Comment by stv0g** [comment]
I am just curious about the existing PR #22.

Are you still planning to finish/merge it?

**Comment by m-mirz** [comment]
Yes, we should merge it. It looks like it is ready. I won't be able to
check it before the weekend. So feel free to add your review :-)
